ocrd_network/ocrd_network/rabbitmq_utils/consumer.py

```diff
@@ -25,9 +24,6 @@ def __init__(self, host: str = HOST, port: int = PORT, vhost: str = VHOST,
         if not logger_name:
             logger_name = __name__
         logger = logging.getLogger(logger_name)
```

Comment:
```suggestion
        logger = getLogger('ocrd_network.rabbitmq_utils.consumer')
```

I should take care of that and test if the logging happens properly though - since the `logger_name` comes from the constructor. However, we could and should remove the `logger_name` parameter as well, since we have a separate file logging for different modules now.

Reply:
Agree, feel free to commit this and similar changes to this branch.